scripts/teamDataScript.js

```diff
@@ -239,17 +239,17 @@ function showAllPitDataTable() //shows pit stats of all teams for the current ev
     var event = getCurrEvent();
     var cname = season + ' pitQuestions';
     var questionsArr = getCookie(cname).split(COOKIE_QUESTION_SEPARATOR).slice(0,-1); //first element to second to last element of .split arr
-    $('#allTeamsPitAnswers').html('');
+    $('#allTeamsPitAnswers-Body').html('');
     $('#allPitAnswersTitle').text(`All Pit Scouting Data for the ${event} event.`);
 
-    var tableHeader = '<tr><th class="allTeamsRow">Team #</th>'; //top-left most cell
+    var widthString = `s${parseInt(12/(questionsArr.length+1))}`; //+1 for the Team# column
+    var tableHeader = `<div class="w3-col ${widthString} w3-center"> <h4>Team #</h4> </div>`; //top-left most cell
+    
```

Comment:
I'm confused, I thought we were switching back **to** `<table>` because it would handle extending horizontally better . . .

Reply:
I think its outdated. Does it look good to merge the PR?